syntax = "proto3";

package proto;

enum Signal{
    START = 0;
    QUIT = 1;
}

message SignalMessage{
    Signal signal = 1;
}